includes/rest-api.php

```diff
@@ -603,6 +683,80 @@ function pmpro_rest_api_get_checkout_levels( $request ) {
 			return new WP_REST_Response( $r );
 		}
 
+
+		/// ZAPIER TRIGGERS
+		/**
+		 * Function used for testing authentication through Zapier API.
+		 * @since 2.7
+		 */
+		public function validate_me( $request ) {
+
+			$params = $request->get_params();
+
+			if ( is_user_logged_in() ) {
+			  $me = wp_get_current_user()->display_name;
+			} else {
+				$me = false;
+			}
+		
+			wp_send_json_success( array( 'username' => $me ) );
+		}
+
+
+		function pmpro_rest_api_recent_memberships( $request ) {
+			$params = $request->get_params();
+
+			$response_type = isset( $params['response_type'] ) ? sanitize_text_field( $params['response_type'] ) : null;
+			$limit = apply_filters( 'pmpro_trigger_recent_members_limit', 1 );
+
+			// Grab the useful information.
+			global $wpdb;
+			$SQL = "SELECT mu.user_id, u.user_email, u.user_nicename, mu.membership_id, ml.name as membership_name, mu.status, mu.modified FROM $wpdb->pmpro_memberships_users AS mu
+					LEFT JOIN $wpdb->users AS u
+					ON mu.user_id = u.id
+					LEFT JOIN $wpdb->pmpro_membership_levels AS ml
+					ON ml.id = mu.membership_id
+					WHERE mu.status = 'active' 
+					ORDER BY mu.modified DESC 
+					LIMIT " . intval( $limit );
+			$results = $wpdb->get_results( $SQL );
```

Comment:
Formatted SQL for easier reading and using wpdb::prepare for including vars in query.

```suggestion

			$sql = "
				SELECT
					`mu`.`user_id`,
					`u`.`user_email`,
					`u`.`user_nicename`,
					`mu`.`membership_id`,
					`ml`.`name` as membership_name,
					`mu`.`status`,
					`mu`.`modified`
				FROM `{$wpdb->pmpro_memberships_users}` AS `mu`
				LEFT JOIN `{$wpdb->users}` AS `u`
					ON `mu`.`user_id` = `u`.`id`
				LEFT JOIN `{$wpdb->pmpro_membership_levels}` AS `ml`
					ON `ml`.`id` = `mu`.`membership_id`
				WHERE
					`mu`.`status` = 'active' 
				ORDER BY
					`mu`.`modified` DESC
				LIMIT %d
			";

			$results = $wpdb->get_results( $wpdb->prepare( $sql, $limit ) );
```